app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -171,6 +171,11 @@ class LoginFragment : Fragment() {
         smartAuthViewModel.requestCredentials(activity)
     }
 
+    private fun handleProgressBar(show: Boolean, dialog: Dialog) {
```

Comment:
No need of this function

Reply:
Create an extension function on Dialog `show(boolean) which does this

Reply:
Okay

Reply:
Done